src/applications/vaos/services/appointment/index.js

```diff
@@ -77,18 +77,29 @@ export async function getAppointmentRequests({ startDate, endDate }) {
  */
 export function isVideoAppointment(appointment) {
   return (
-    appointment.contained?.some(
-      contained =>
-        contained.resourceType === 'HealthcareService' &&
-        contained.characteristic?.some(
-          c =>
-            c.coding === VIDEO_TYPES.gfe ||
-            c.coding === VIDEO_TYPES.videoConnect,
-        ),
-    ) || false
+    appointment.contained
+      ?.find(contained => contained.resourceType === 'HealthcareService')
+      ?.characteristic?.some(c =>
+        c.coding?.some(code => code.system === 'VVS'),
```

Comment:
`coding` is an array in the FHIR format, so I tweaked that in the transformer and updated these functions to reflect it.